src/app/Http/Controllers/Admin/UniquePageCrudController.php

```diff
@@ -0,0 +1,245 @@
+<?php
+
+namespace Backpack\PageManager\app\Http\Controllers\Admin;
+
+use App\UniquePages;
+use Illuminate\Http\Request;
+use Backpack\PageManager\app\TraitReflections;
+use Backpack\CRUD\app\Http\Controllers\CrudController;
+use Backpack\CRUD\app\Http\Controllers\CrudFeatures\SaveActions;
+
+class UniquePageCrudController extends CrudController
+{
+    use SaveActions;
+    use UniquePages;
+    use TraitReflections;
+
+    public function setup()
+    {
+        parent::__construct();
+
+        $modelClass = config('backpack.pagemanager.unique_page_model_class', 'Backpack\PageManager\app\Models\Page');
+
+        $this->checkForTemplatesAndUniquePagesNotDistinct();
+
+        /*
+        |--------------------------------------------------------------------------
+        | BASIC CRUD INFORMATION
+        |--------------------------------------------------------------------------
+        */
+        $this->crud->setModel($modelClass);
+        // Don't set route or entity names here. These depend on the page you are editing
+
+        // unique pages cannot be created nor deleted
```

Comment:
or listed?

Reply:
Actually what is the point of this comment?

Reply:
wanted to inform why we deny this access...
what do you think? remove the comment entirely?